Bottom: e36714b38d388624c6b5c5cf5e83385ba9c559e0
Top:    74c916967de618742f5df336b6c6aca1160c120d
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 15:34:01 +0800

vhost_net: add interrupt coalescing support

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 7d137a4..9740534 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -415,6 +415,7 @@ static void handle_tx(struct vhost_net *net)
 		}
 	}
 out:
+	vhost_check_coalesce_and_signal(vq->dev, vq);
 	mutex_unlock(&vq->mutex);
 }
 
@@ -638,6 +639,7 @@ static void handle_rx(struct vhost_net *net)
 		}
 	}
 out:
+	vhost_check_coalesce_and_signal(vq->dev, vq);
 	mutex_unlock(&vq->mutex);
 }
